Utilities/DataLoading/EdFi.BulkLoadClient.Console/Application/CommandLineOverrides.cs

```diff
@@ -47,8 +50,8 @@ public class CommandLineOverrides
         [Option('w', "working", Required = false, HelpText = "Path to a writable folder containing the working files")]
         public string WorkingFolder { get; set; }
 
-        [Option('x', "xsd", Required = false, HelpText = "(deprecated) Path to a folder containing the Ed-Fi Xsd Schema files")]
-        public string XsdFolder { get; set; }
+        [Option('x', "xsdmetadataurl", Required = false, HelpText = "(deprecated) The XSD metadata url (i.e. http://server/metadata)")]
+        public string XsdMetadataUrl { get; set; }
```

Comment:
I don't remember the details right now, but I have had trouble with the bulk client downloading XSD, and I typically just rely on the folder. Besides, the URL should be available on the base `/api` endpoint.

Reply:
@stephenfuqua I was able to reliably download XSD files from the API endpoint after I implemented the fix in this pull request. 

@vimayya MetadataUrl and XsdMetadataUrl are both used in the source code but only MetadataUrl can be overridden on the command line.  Because the -x "xsd" command line argument is marked as deprecated, I "reused" it for XsdMetadataUrl.  I can create a new command line argument for it... would that suffice?

Reply:
There are more places that have to be updated to completely remove XsdFolder. We will handle it in a different ticket.
For now, can you bring it back and use a different letter for XsdMetadataUrl?

Reply:
@AxelMarquez @vimayya Both XsdFolder and XsdMetadataUrl are available in command line arguments.